bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
Actually it could be even `int` though. It is `unsigned` due to being `size` but not `index`. What this function does; I don't see any advantage of using `unsigned int` explicitly. @tcare already pointed out uint64_t part.